package.json

```diff
@@ -15,8 +15,8 @@
     "test": "cross-env npm run testall || npm run testall",
     "coveralls": "nyc --instrument=false --source-map=false report --temp-directory=./coverage --reporter=text-lcov | coveralls",
     "precommit": "lint-staged",
-    "lint": "lerna run lint",
-    "prepublish": "lerna run prepublish"
+    "lint": "standard",
+    "prepublish": "npm run lint"
```

Comment:
This seems incorrect.

Reply:
And it's the reason the tests are breaking 🕵️

Reply:
🤭 good catch!